apps/web/src/components/Sidebar/index.tsx

```diff
@@ -1,22 +1,17 @@
-import { HEAD_COMMIT, materializeDocumentsAtCommit } from '@latitude-data/core'
+import {
+  HEAD_COMMIT,
+  listCommits,
+  materializeDocumentsAtCommit,
+} from '@latitude-data/core'
 
-import DocumentTree, { CreateNode } from './DocumentTree'
+import DocumentTree from './DocumentTree'
 
 export default async function Sidebar() {
-  const documents = await materializeDocumentsAtCommit({
+  const docsResult = await materializeDocumentsAtCommit({
     commitUuid: HEAD_COMMIT,
-    staged: true,
   })
 
-  return (
-    <div className='flex flex-col gap-4 p-4'>
-      <div className='flex flex-row align-items justify-between'>
-        <h2>Prompts</h2>
-        <div className='flex flex-row gap-2 align-items'>
-          <CreateNode />
-        </div>
-      </div>
-      <DocumentTree documents={documents} />
-    </div>
-  )
+  const commits = await listCommits()
```

Comment:
it's puzzling that read queries are not wrapped in result but writes are, we have to wrap all in result

Reply:
I hard agree on this one